From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kenneth Myhra <kennethmyhra@gmail.com>
Date: Thu, 12 Aug 2021 20:59:25 +0200
Subject: [PATCH] ntohl/ntohs is located in 'arpa/inet.h'

In Serenity ntohl/ntohs is located in arpa/inet.h, other stuff glib
needs is included in 'netinet/in.h'.
---
 gio/xdgmime/xdgmimecache.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/gio/xdgmime/xdgmimecache.c b/gio/xdgmime/xdgmimecache.c
index 7e9b29fadb33c2960eb3981753c94dc41ed3b0bf..8fb32200651265a357835fd70e27f5ae8cd73ee3 100644
--- a/gio/xdgmime/xdgmimecache.c
+++ b/gio/xdgmime/xdgmimecache.c
@@ -22,7 +22,12 @@
 #include <fnmatch.h>
 #include <assert.h>
 
+#ifdef __serenity__
+#include <arpa/inet.h>  /* for ntohl/ntohs */
+#include <netinet/in.h>
+#else
 #include <netinet/in.h> /* for ntohl/ntohs */
+#endif
 
 #ifdef HAVE_MMAP
 #include <sys/mman.h>
